kotlin/workflow-testing/src/main/java/com/squareup/workflow/testing/RealRenderTester.kt

```diff
@@ -213,6 +213,29 @@ internal class RealRenderTester<PropsT, StateT, OutputT : Any, RenderingT>(
     return expected
   }
 
+  private inline fun <reified T : Expectation<*>> consumeExpectedWorker(
```

Comment:
My kotlin is weak here i think i can define this more explicitly `private inline fun <reified T : ExpectedWorker<*>> consumeExpectedWorker`